.github/workflows/main.yml

```diff
@@ -28,7 +30,11 @@ jobs:
     - name: Config
       id: cfg
       run: |
-        if "${GITHUB_REF}" in "refs/tags/"*; then
+        push_test_pypi=no
+        push_pypi=no
+
+        if [[ "${GITHUB_REF}" =~ refs/tags/.* ]]; then
+          echo "push_test_pypi=yes" >> $GITHUB_OUTPUT
```

Comment:
We're not using these two settings separately, either we push to both test and real pypi, or we do neither.

Reply:
Yeah, I thought I'd leave them both in just in case we want to introduce an additional trigger condition for test pypi. It can be refined up with the next update to the workflow I suppose